pkg_defs/aca_view/meta.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: aca_view
+  version:  {{ SKA_PKG_VERSION }}
+
+build:
+  noarch: python
+  script_env:
+    - SKA_TOP_SRC_DIR
```

Comment:
Probably not needed, but if you've built the package with this for 2020.4rc1 can leave it?